src/cuda/common.cu

```diff
@@ -164,40 +163,39 @@ void onedim_fseries_kernel_precomp(CUFINUFFT_BIGINT nf, T *f, std::complex<doubl
   for (int n = 0; n < q; ++n) {                           // set up nodes z_n and vals f_n
     z[n] *= J2;                                           // rescale nodes
     f[n] = J2 * w[n] * evaluate_kernel((T)z[n], opts);    // vals & quadr wei
-    a[n] = exp((T)(2.0 * M_PI) * std::complex<T>(0.0, 1.0) * (T)(nf / 2 - z[n]) /
-               (T)nf);                                    // phase winding rates
+    a[n] = ((T)(2.0 * M_PI) * (T)(nf / 2 - z[n]) / (T)nf); // phase winding rates
```

Comment:
@ahbarnett 
Together with @blackwer we figured out that we could not do the exp here and keep the polar angle.